generative/inferers/inferer.py

```diff
@@ -303,11 +304,26 @@ class LatentDiffusionInferer(DiffusionInferer):
         scheduler: a scheduler to be used in combination with `unet` to denoise the encoded image latents.
         scale_factor: scale factor to multiply the values of the latent representation before processing it by the
             second stage.
+        ldm_latent_shape: desired SPATIAL latent space shape. Used if there is a difference in output VAE latent shape
+        and LDM shape.
+        vae_latent_shape: VAE SPATIAL latent space shape. Used if there is a difference in output VAE latent shape and
```

Comment:
```suggestion
        autoencoder_latent_shape: autoencoder spatial latent space shape. Used if there is a difference between the autoencoder's latent shape and
```

Reply:
we call it an autoencoder in the inferer, and should avoid calling it a VAE specifically because the model we use is not always a VAE